docs/project-list.md

```diff
@@ -0,0 +1,103 @@
+---
+date-generated: 2020-03-24T14:44:10
+tool: NuGetTasks.GenerateMarkdownDoc
+---
+
+
+# NuGet Project Overview
```

Comment:
I'd also call the file "project-overview.md"

Reply:
Changed to project-overview.md